messaging/src/main/java/org/axonframework/commandhandling/distributed/DistributedCommandBus.java

```diff
@@ -347,6 +335,19 @@ public Builder defaultCommandCallback(CommandCallback<Object, Object> defaultCom
             return this;
```

Comment:
You need to validate that the given `defaultCommandBack` is non-null here.

Reply:
Add a null check